geometry_msgs/Point[3] Triangle_Vertices
